bigquery/src/main/java/com/google/cloud/hadoop/io/bigquery/samples/WikipediaRequestBytes.java

```diff
@@ -34,17 +34,10 @@
 
 /**
  * Sample MapReduce which computes SUM(bytes_transferred) grouping by "title" from a
- * wikipedia_pageviews table. The following is the BigQuery schema of the table:
- *   datetime_epoch INTEGER NULLABLE
- *   year INTEGER NULLABLE
- *   month INTEGER NULLABLE
- *   day INTEGER NULLABLE
- *   hour INTEGER NULLABLE
- *   wikimedia_project STRING NULLABLE
- *   language STRING NULLABLE
- *   title STRING NULLABLE
- *   views INTEGER NULLABLE
- *   bytes_transferred INTEGER NULLABLE
+ * wikipedia_pageviews table. The following is the BigQuery schema of the table: datetime_epoch
+ * INTEGER NULLABLE year INTEGER NULLABLE month INTEGER NULLABLE day INTEGER NULLABLE hour INTEGER
```

Comment:
Revert, we need to fix javadoc instead.

Reply:
How about fixing javadoc in a separate PR ?

Reply:
Sure, but let's submit that PR first.

Reply:
will create a PR today